libraries/AP_InertialSensor/AP_InertialSensor_Backend.h

```diff
@@ -123,6 +123,7 @@ class AP_InertialSensor_Backend
         DEVTYPE_INS_ICM42605 = 0x35,
         DEVTYPE_INS_ICM40605 = 0x36,
         DEVTYPE_INS_IIM42652 = 0x37,
+        DEVTYPE_BMI270       = 0x38,
```

Comment:
need to add to Tools/scripts/decode_devid.py

Reply:
Fixed